index.d.ts

```diff
@@ -1,6 +1,6 @@
 declare function MMKVStorage(): any;
 
-export declare function useMMKVStorage(key:string,storage:MMKVStorage.API):[value:any,setValue:(value?:any)=>void];
+export declare function useMMKVStorage(key: string, storage: MMKVStorage.API): [any, (value?: any) => void];
```

Comment:
How about using generics to declare the output type ?

```suggestion
export declare function useMMKVStorage<T extends any>(key: string, storage: MMKVStorage.API): [T, (value?: T) => void];
```

Reply:
That works for me! Seems to compile fine for me this way as well.